SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -445,12 +447,12 @@ void CIncNSSolver::BC_Wall_Generic(const CGeometry *geometry, const CConfig *con
     }
     else { // ISOTHERMAL
 
-      auto Point_Normal = geometry->vertex[val_marker][iVertex]->GetNormal_Neighbor();
+      const auto Point_Normal = geometry->vertex[val_marker][iVertex]->GetNormal_Neighbor();
 
       /*--- Get coordinates of i & nearest normal and compute distance ---*/
 
-      auto Coord_i = geometry->nodes->GetCoord(iPoint);
-      auto Coord_j = geometry->nodes->GetCoord(Point_Normal);
+      const su2double* Coord_i = geometry->nodes->GetCoord(iPoint);
+      const su2double* Coord_j = geometry->nodes->GetCoord(Point_Normal);
```

Comment:
put auto back

Reply:
I guess for all `su2double*`

Reply:
Yes, in case we ever start returning an iterator or something less restrictive than a pointer.